app/models/recruitment_cycle_timetable.rb

```
@@ -54,16 +54,14 @@ def cycle_range
   end
 
   def sequential_dates
-    required_dates = [
+    return if [
```

I wish there was a way in Rails to order the validations and fail fast so we didn't have to do this.